docs/getting-started/getting-started-with-gossip-ring.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Getting Started with Gossiped Ring (experimental)"
+linkTitle: "Gossip Ring (experimental)"
+weight: 2
+slug: getting-started-with-gossiped-ring
+---
+
+Cortex requires Key-Value store to store the ring. It can use traditional KV stores like Consul or Etcd,
+but now it can also build its own KV store on top of memberlist library using a gossip algorithm.
+
+This short guide shows how to start Cortex in [single-binary mode](../architecture.md) with memberlist-based ring.
+To reduce number of required dependencies in this guide, it will use [blocks storage](../operations/blocks-storage.md) with no shipping to external stores.
+Storage engine and external storage configuration are not dependant on the ring configuration.
+
+## Single-binary, two Cortexes
+
+
+
+For simplicity and to get started, we'll run it as a two instances of Cortex on local computer.
+We will use prepared configuration files ([file 1](../configuration/single-process-config-blocks-gossip-1.yaml), [file 2](../configuration/single-process-config-blocks-gossip-2.yaml)), with no external
+dependencies.
+
+Build Cortex first:
+```sh
+$ go build ./cmd/cortex
+```
+
+First instance:
+```
+$ ./cortex -config.file docs/configuration/single-process-config-blocks-gossip-1.yaml
+```
+
+Second instance is similar, but uses `...-2.yaml` file.
+```
+$ ./cortex -config.file docs/configuration/single-process-config-blocks-gossip-2.yaml
+```
+
+The two instances should be able to find each other via memberlist configuration:
```

Comment:
To a first time reader, it may not be clear if the following config is a change they should do or it's just you explaining how things work under the hood. An alternative may be explaining how to run Cortex in this section and then adding another section below (ie. "How it works") explaining how things work under the hood. In this way we keep the two things separated.